vector/src/Data/Vector/Primitive.hs

```diff
@@ -1805,3 +1829,6 @@ unsafeCopy = G.unsafeCopy
 copy :: (Prim a, PrimMonad m) => MVector (PrimState m) a -> Vector a -> m ()
 {-# INLINE copy #-}
 copy = G.copy
+
+-- $setup
+-- >>> default (Int)
```

Comment:
See my comment above. This will cause trouble for non experienced Haskellers. And I don't want to answer github tickets: "The doctests don't work" :smile: